tests/sentry/workflow_engine/migrations/test_0054_clean_up_orphaned_metric_alert_objects.py

```diff
@@ -0,0 +1,54 @@
+from sentry.testutils.cases import TestMigrations
+from sentry.workflow_engine.migration_helpers.alert_rule import dual_write_alert_rule
+from sentry.workflow_engine.models import (
+    Action,
+    ActionAlertRuleTriggerAction,
+    AlertRuleDetector,
+    AlertRuleWorkflow,
+    DataCondition,
+    DataConditionGroup,
+    DataConditionGroupAction,
+)
+
+
+class TestCleanUpOrphanedMetricAlertObjects(TestMigrations):
```

Comment:
Let's comment this out now that it has proved this runs fine, just to make sure ci doesn't end up breaking.